src/datta_lab_to_nwb/markowitz_gillis_nature_2023/markowitz_gillis_nature_2023_convert_session.py

```diff
@@ -57,12 +56,22 @@ def session_to_nwb(data_dir_path: Union[str, Path], output_dir_path: Union[str,
 
 if __name__ == "__main__":
     # Parameters for conversion
-    data_dir_path = Path("/Directory/With/Raw/Formats/")
-    output_dir_path = Path("~/conversion_nwb/")
+    file_path = Path(
+        "/Volumes/T7/CatalystNeuro/NWB/Datta/dopamine-reinforces-spontaneous-behavior/dlight_raw_data/dlight_photometry_processed_full.parquet"
+    )
+    metadata_path = Path(
+        "/Volumes/T7/CatalystNeuro/NWB/Datta/dopamine-reinforces-spontaneous-behavior/dlight_raw_data/session_metadata.yaml"
+    )
+    output_dir_path = Path("/Volumes/T7/CatalystNeuro/NWB/Datta/conversion_nwb/")
+    shutil.rmtree(output_dir_path)
```

Comment:
Just out of curiosity, why did you include `rmtree` here?

Reply:
I included `rmtree` so that I could rerun the conversion without having to manually delete the folder

Reply:
OK I guess - just so you know, there is also a keyword to `run_conversion` called `overwrite`, and if you set `overwrite=True` it will automatically create a new file at the same location